app/api/osu/web.py

```diff
@@ -606,7 +606,82 @@ async def submit_score_handler(
 
     # TODO: construct score submission charts
 
-    return b"error: no"
+    response_data = bytearray()
+
+    # TODO: fetch real values
+    beatmap_rank_before = 0
+    beatmap_rank_after = 0
+    beatmap_ranked_score_before = 0
+    beatmap_ranked_score_after = 0
+    beatmap_total_score_before = 0
+    beatmap_total_score_after = 0
+    beatmap_max_combo_before = 0
+    beatmap_max_combo_after = 0
+    beatmap_accuracy_before = 0.0
+    beatmap_accuracy_after = 0.0
+    beatmap_pp_before = 0.0
+    beatmap_pp_after = 0.0
+
+    # TODO: fetch real values
+    overall_rank_before = 0
+    overall_rank_after = 0
+    overall_ranked_score_before = 0
+    overall_ranked_score_after = 0
+    overall_total_score_before = 0
+    overall_total_score_after = 0
+    overall_max_combo_before = 0
+    overall_max_combo_after = 0
+    overall_accuracy_before = 0.0
+    overall_accuracy_after = 0.0
+    overall_pp_before = 0.0
+    overall_pp_after = 0.0
+
+    response_data += (
+        f"beatmapId:{beatmap['beatmap_id']}|"
+        f"beatmapSetId:{beatmap['beatmap_set_id']}|"
+        f"beatmapPlaycount:{beatmap['plays']}|"
+        f"beatmapPasscount:{beatmap['passes']}|"
+        # TODO: is this correct?
+        f"approvedDate:{beatmap['bancho_updated_at'].isoformat()}|"
```

Comment:
this part shows like "{playcount} plays since {approved_date}", so i think since our server has it's own measurement of playcount & plays, it makes most sense to actually keep this value to our `created_at`
```suggestion
        f"approvedDate:{beatmap['created_at'].isoformat()}|"
```